sdk/cognitiveservices/Azure.ResourceManager.CognitiveServices/src/Generated/BaseCognitiveServicesAccountResource.cs

```diff
@@ -0,0 +1,154 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// <auto-generated/>
+
+#nullable disable
+
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure;
+using Azure.Core;
+using Azure.ResourceManager;
+using Azure.ResourceManager.Resources;
+
+namespace Azure.ResourceManager.CognitiveServices
+{
+    /// <summary> This is the base client representation of the following resources <see cref="CognitiveServicesAccountResource" /> or <see cref="CognitiveServicesDeletedAccountResource" />. </summary>
+    public abstract partial class BaseCognitiveServicesAccountResource : ArmResource
```

Comment:
We need a different prefix other than `Base`.

Reply:
Would `CognitiveServicesAccountBaseResource` or `BasicCognitiveServicesAccountResource` be better in this case?